src/features/farming/crops/components/Soil.tsx

```diff
@@ -80,6 +81,13 @@ export const Soil: React.FC<Props> = ({
 
     return (
       <div className="relative w-full h-full">
+        {plantedCrop?.fertilisers && (
+          <div className="flex absolute z-10 right-1 -top-2">
```

Comment:
I did a bit of playing, not sure what you think. 
```suggestion
          <div className="flex-col absolute z-10 - right-1 -top-2">
```
It moves the fertiliser out a little bit, and puts them in a column so they dont overlap the crop too much.

![fertiliser](https://user-images.githubusercontent.com/41215134/184041967-cd548948-334a-482d-832b-749ca5a5b8e6.png)